app/services/process_submission_service.rb

```diff
@@ -99,7 +112,7 @@ def attachments(mail)
       object['path'] = url_file_map[object['url']]
       object
     end
-    array.map{|o| Attachment.new(o.symbolize_keys)}
+    array.map { |o| Attachment.new(o.symbolize_keys) }
```

Comment:
Undo

Reply:
is there a code convention yet? could we make a rubo cop file to enforce a syle for this?